src/resource.ts

```diff
@@ -262,13 +263,16 @@ export default class Resource<T = any> {
    * The rel argument is optional. If it's given, we will only return links
    * from that relationship type.
    */
-  async links(rel?: string): Promise<Link[]> {
+  async links(rel?: string, anchor?: string): Promise<Link[]> {
```

Comment:
This is a bit I'm a little uncertain of. Should it filter by default? Does it make sense to ask for links for (potentially) a different resource? Should resources with fragements be treated as separate resources? I really don't know what the right answer is, and I'm curious what your thoughts are.

Would it be sufficient (for now) to not filter links at all and just return all links (with anchor or without)?